src/components/NavigationLayout/NavigationLayout.tsx

```diff
@@ -59,66 +59,63 @@ export const NavigationLayout: React.FC<NavigationLayoutProps> = ({
-                                <LibraryVersion id={section.id} />
-                                {subSection ? (
-                                    <span className={b('mobile-navigation-control-sub-section')}>
-                                        {' '}
-                                        • {subSection.title}
-                                    </span>
-                                ) : null}
-                            </div>
-                            <div className={b('mobile-navigation-control-arrow')}>
-                                <Icon data={arrowIcon} width={10} height={6} />
-                            </div>
+                            <Icon data={menuCloseIcon} width={16} />
                         </div>
-                        <div className={b('navigation', {'mobile-open': isOpenMobileNavigation})}>
-                            <div className={b('mobile-navigation-header')}>
-                                <div className={b('mobile-navigation-header-title')}>
-                                    {mobileTitle}
-                                </div>
-                                <div
-                                    tabIndex={0}
-                                    role="button"
-                                    className={b('mobile-navigation-header-close')}
-                                    onClick={() => {
-                                        setIsOpenMobileNavigation(false);
-                                    }}
-                                >
-                                    <Icon data={menuCloseIcon} width={16} />
-                                </div>
-                            </div>
-                            <Navigation
-                                sections={sections}
-                                sectionId={sectionId}
-                                subSectionId={subSectionId}
-                                searchPlaceholder={searchPlaceholder}
-                                emptySearchPlaceholder={
-                                    emptySearchPlaceholder ?? t('emptySearchPlaceholder')
-                                }
-                                onClickOnLink={clickOnLinkHandler}
-                            />
-                        </div>
-                    </Col>
-                    <Col sizes={{all: 12, lg: 9}}>
-                        <div className={b('content')}>{children}</div>
-                    </Col>
-                </Row>
-            </Grid>
+                    </div>
+                    <Navigation
+                        sections={sections}
+                        sectionId={sectionId}
+                        subSectionId={subSectionId}
+                        searchPlaceholder={searchPlaceholder}
+                        emptySearchPlaceholder={
+                            emptySearchPlaceholder ?? t('emptySearchPlaceholder')
+                        }
+                        onClickOnLink={clickOnLinkHandler}
```

Comment:
Is it possible to keep the "scroll to top when clicking on menu item" behavior?

https://github.com/user-attachments/assets/0585320b-4b2d-446c-b5c6-c8c6060fc396